reddit_liveupdate/templates/liveupdateeventapp.html

```diff
@@ -37,23 +42,19 @@ <h1 id="liveupdate-title">${thing.event.title}</h1>
 <div class="main-content">
 % if thing.show_sidebar:
 <aside class="sidebar side md-container">
-  % if thing.event.description:
-  <section id="liveupdate-description">
-    ${utils.md(thing.event.description, wrap=True)}
-  </section>
-  % endif
+  <a class="sidebar-expand" href="javascript: void 0;" data-toggle="${_("less about this live thread")}">${_("more about this live thread")}</a>
```

Comment:
I did see that but felt this way keeps the related strings closer together in the code.